pkg/controllers/raycluster_controller.go

```diff
@@ -179,9 +183,10 @@ func (r *RayClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			}
 		}
 
-	} else if cluster.Status.State != "suspended" && !r.isRayDashboardOAuthEnabled() && !isOpenShift {
+	} else if cluster.Status.State != "suspended" && !r.isRayDashboardOAuthEnabled() && !r.IsOpenShift {
+		ingressDomain := ""
```

Comment:
Can you add a TODO notifier to fix providing Ingress domain?
Together with link to JIRA referencing the addition of Ingress domain.